helpers.h

```diff
@@ -1,16 +1,19 @@
-// To be Included the header file "bmp.h"
+#include "bmp.h"
+#include <vector>
+using namespace std;
```

Comment:
Please remove using namespace `std` because you are using `std` everywhere even if you are including `std`.